### ISSUE TITLE
HTTP Error 403: The user '' isn't allowed to upload to project ''

### ISSUE THREAD

**Issue by i2y** [pypa/warehouse#11610]
I'm facing on the same/similar issue with [this issue](https://github.com/pypa/warehouse/issues/11013).
I tried to publish [my python package "hippos"](https://github.com/i2y/hippos) to PyPI, but it failed with the error of this PR title even though that name doesn’t seem to exist in PyPI. Could you please help me?🙏

Btw, I actually first tried to register using the name "cat", but that gave me the same error.  And really, if I can use ”cat”,  I might want to use "cat".  Anyways, sorry, for now, I'm happy if I would be able to register it under the name "hippos" first 🙇 (For now, I already had registered under the name "Cattt" because I needed to register in a hurry.)

**Comment by i2y** [comment]
ah.. sorry, I just realized I misunderstood `cat`  doesn't exist in PyPI. [There is `cat` in PyPI.](https://pypi.org/project/cat/)
On the other hand, `hippos` doesn't seem to really exist in PyPI at this time at the least. (https://pypi.org/project/hippos/ -> HTTP 404)

**Comment by i2y** [comment]
I just found the another similar issue from already closed issues.
https://github.com/pypa/warehouse/issues/10139

I saw that in that Issue above the person was told to open [PEP 541 request on pypi-support](https://github.com/pypa/pypi-support/issues/new?labels=PEP+541&template=pep541-request.yml&title=PEP+541+Request%3A+PROJECT_NAME).
However, both the above issue and mine are not name conflict issues but the unavailability of a name that does not appear to exist in PyPI. Even if so, should I open the PEP 541 request form?
If I should, and if doing so would solve my issue quickly, I would like to do so.
I would appreciate, the authorized person, could give me your opinion and instructions.

Maybe in a few more hours, I may start thinking of another name (because I want to get my package naming issue done by the end of this weekend...).  But, I have already spent a lot of time trying to come up with a name for my package... I'm not sure if I will come up with a new name soon.
